### ISSUE TITLE
Pin: How to get blender-rhubarb-lipsync working on the latest version of Blender (including 3.6)

### ISSUE THREAD

**Issue by KaijuKoder** [scaredyfish/blender-rhubarb-lipsync#63]
To all those people who keep posting here saying this doesn't work on the latest version of Blender:

**Long version**

**Bad news:** Unfortunately @scaredyfish / blender-rhubarb-lipsync seems abandoned.

**Good news:** @Premik has ported it to the latest version of Blender which I can confirm works as of 3.6: https://github.com/Premik/blender_rhubarb_lipsync_ng  (ng = next generation?)

**Bad news:** Unfortunately @Premik's version doesn't support linked-libraries - though it seems to work with everything else.

**Good news:** I worked out how to do this, and posted here: https://github.com/Premik/blender_rhubarb_lipsync_ng/issues/7

**Bad news:** @Premik hasn't incorporated those changes into the new version yet. I hope this new version isn't abandoned too.

**Short version**

1. Go to https://github.com/Premik/blender_rhubarb_lipsync_ng/releases. Get the latest version. Install. Read the notes because the interface is different and better.
2. A fix for linked-libraries is in https://github.com/Premik/blender_rhubarb_lipsync_ng/issues/7 but best wait for @Premik to return and check it.

**Comment by okikiola12** [comment]
a video tutorial can be handy here, the interface is different you know

**Comment by Premik** [comment]
>>a video tutorial can be handy here, the interface is different you know

Yeah. But for a quick start the [project readme](https://github.com/Premik/blender_rhubarb_lipsync_ng/blob/master/README.md) contains some screenshots and screen captures.

**Comment by KaijuKoder** [comment]
Again, to all those people who keep posting here saying this doesn't work on the latest version of Blender:

@Premik has updated https://github.com/Premik/blender_rhubarb_lipsync_ng (next generation?) which not only works with Blender 4.0 but also works with library linked rigs (i.e. overrides). Lovely! Rhubarb is back! :-)

**Comment by okikiola12** [comment]
yeah working with no issues

**Comment by IhorSavaryn** [comment]
it calculating but can't create cues and freeze without error. (blender 4.0)

**Comment by IhorSavaryn** [comment]
so, it works with text/audio track no more 5-10 sec, it is not cool

**Comment by okikiola12** [comment]
can you be more specific? 
if you are talking about the keyframe. then it is in development by the dev.
I understand that without it, interpolation and other f-curve manipulation is not possible with NLA.
again in the meantime, check your NLA if you want to edit.

check this tutorial.

https://odysee.com/@OwlGear:8/RhubarbLipsyncNGBlenderplugin-fromScratch:7

**Comment by Premik** [comment]
> so, it works with text/audio track no more 5-10 sec, it is not cool

@IhorSavaryn There was this problem reported before, but I believe it has been fixed in `v1.0.2+`. Did you perhaps used the `v1.0.1`?  More details here: https://github.com/Premik/blender_rhubarb_lipsync_ng/issues/8#issuecomment-1753829490

**Comment by okikiola12** [comment]
@Premik PERFECT. but happens to the auto blend in & out.
it looks as if it has been phased out in the latest v1.2.1
